interface/grpc/server.go

```diff
@@ -20,7 +21,8 @@ import (
 
 // Server contains the server config.
 type Server struct {
-	api *api.API
+	container container.Container
```

Comment:
I don't think this is good. The API already has the container so there is no reason for the Server to have it too.
2 solutions: 
- The server has no container information and we need a `Status` function in the API 👍 
- The server only has the container and not the api and the server give the container to the api when executing if needed. We could have different container system configurable from the api

I would definitely choose the first one as the second we don't have multiple containers system and i'm not even sure we will want to configure it from the api

Reply:
yes i agree. this is something that we'll refactor in future. either by going with the option 1 of yours or have `grpc.GetService()` to return status by default or optionally. let's do not add more to this PR.

Reply:
I think this has to be done in this PR.
This PR is adding an unnecessary dependency that actually adds even more review in the code.
With the PR https://github.com/mesg-foundation/core/pull/858 many files from this PR will be untouched or the same piece of code will be changed anyway.
I really don't want to add a new dependency in this grpc api that is not necessary just for one single function that should anyway be in the api